docs/internals/contributing/writing-documentation.txt

```diff
@@ -573,6 +577,41 @@ you'd like to help translate the documentation into another language.
 
 .. _django-admin-manpage:
 
+Accessibility in documentation
+==============================
+
+See :doc:`/internals/contributing/writing-code/accessibility` for our project guidelines. For documentation
+specifically, guide readers to use accessible patterns.
+
+* Include enough code so examples are accessible by default. For example:
+
+  * :class:`django.forms.ImageField` usage should
+    acknowledge the need for alt text.
```

Comment:
I think for `alt` this makes sense. For others I'm less sure but I'm also less sure if we have many examples that would fall under this.

For example I think adding a bunch of aria attributes to an element might also make it confusing for beginners and are very context-dependent.

Reply:
Good point, I’ll see about producing examples. The way I see it, if you use ARIA attributes it’s because there’s no better way. So if an element didn’t have those attributes even though it needs them – that’s a major issue that would clearly need fixing in the doc, either by using a different element or adding the attributes.